src/index.html

```diff
@@ -12,13 +12,12 @@
     <header>
       <h1><span>fresh</span>Picks<img src="../images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
       <form id="search" role="search">
-        <input id="search-input" type="text" placeholder="Search by Recipe or Ingredient"></input>
+        <input id="search-input" type="text" placeholder="Search Recipe or Ingredient..."></input>
         <label for="search-input" class="search-label">Search</label>
-        <button class="search-btn" type="button" name="button"><img src="../images/search.svg" alt="search button"></button>
+        <button class="nav-btn search-btn" type="button" name="button"><img src="../images/search.png" alt="search button">Search</button>
       </form>
-      <button class="search-btn saved-recipes-btn" type="button" name="button">My Recipes</button>
-      <button class="search-btn my-pantry-btn" type="button" name="button">My Pantry</button>
-
+      <button class="nav-btn ing-pan-btn saved-recipes-btn" type="button" name="button"><img src="../images/cookbook.png" alt="search button">My Recipes</button>
+      <button class="nav-btn ing-pan-btn saved-ingredients-btn" type="button" name="button"><img src="../images/seasoning.png" alt="search button">My Pantry</button>
```

Comment:
We have to add back in that my-pantry class for pantry functionality!

Reply:
Fixed